From: Sven Eckelmann <sven@narfation.org>
Date: Wed, 5 Dec 2018 13:52:01 +0100
Subject: [PATCH] batman-adv: Allow to disable depmod

On some build systems, the actual call to depmod is not expected and only
the installation of the module is expected. Just allow to overwrite this
call with a different (dummy) executable.

Signed-off-by: Sven Eckelmann <sven@narfation.org>
---
 Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index d665c0baf2800957c7659004ad629c0cb185fbc3..08ac8f65e8f51405d4972c9b7f15d3e95c6cd626 100644
--- a/Makefile
+++ b/Makefile
@@ -45,6 +45,7 @@ export KERNELPATH
 RM ?= rm -f
 CP := cp -fpR
 LN := ln -sf
+DEPMOD := depmod -a
 
 REVISION= $(shell	if [ -d "$(PWD)/.git" ]; then \
 				echo $$(git --git-dir="$(PWD)/.git" describe --always --dirty --match "v*" |sed 's/^v//' 2> /dev/null || echo "[unknown]"); \
@@ -90,7 +91,7 @@ clean:
 
 install: config
 	$(MAKE) -C $(KERNELPATH) $(BUILD_FLAGS) modules_install
-	depmod -a
+	$(DEPMOD)
 
 config:
 	$(PWD)/gen-compat-autoconf.sh $(PWD)/compat-autoconf.h
